packages/lab/src/stepper/Stepper.css

```diff
@@ -0,0 +1,41 @@
+.saltStepper {
+  grid-area: stepper;
+  width: 100%;
+  height: 100%;
+
+  padding: 2px;
```

Comment:
these probably should be in function of a token too, but couldn't find a good example 🤔

Reply:
The padding/margin thing is a hack, to make the focus state work, I can use a token there as well